content/docs/guides/components/modeldb.md

```diff
@@ -0,0 +1,94 @@
++++
+title = "ModelDB"
+description ="ModelDB - A system to manage machine learning models"
+weight = 10
+toc = true
+[menu]
+[menu.docs]
+  parent = "components"
+  weight = 5
++++
+## Deploying ModelDB
+
+Use the below commands to deploy ModelDB.
+
+```
+ks generate modeldb modeldb
+ks apply default -c modeldb
+```
+
+## Using ModelDB
+ 
+ModelDB organizes model data in a 3-level model hierarchy. At the bottom of the hierarchy, is an ExperimentRun: every execution of a script/program creates an ExperimentRun.
+Next, related ExperimentRuns can be grouped into an Experiment (e.g., "running hyperparameter optimization for the Neural Network"). Finally, all Experiments and ExperimentRuns belong to a Project (e.g., "churn prediction").
+
+After ModelDB is deployed and modeldb-db, modeldb-backend and modeldb-frontend pods are running - 
+
+1)  Loading Metrics from Python
+
+Open up a Jupyter notebook you will use for model training. Make sure that the ModelDB libraries have been installed already or install via 'pip install modeldb'.
+
+ModelDBSyncer is the object that logs models and operations to the ModelDB backend.
+You can initialize the Syncer with your specified configurations as shown below.
+
+* Create a syncer using a convenience API
+
+```
+syncer_obj = Syncer.create_syncer("Project Name", 
+"test_user", 
+"project description", 
+host="modeldb-backend")
+```
+
+For other ways of initializing the syncer see [here](https://github.com/mitdbg/modeldb/blob/master/client/python/light_api.md#b-create-a-modeldb-syncer).
+
+3)  Sync Information
+
+Initialize the Dataset, Model, ModelConfig, ModelMetrics classes with the needed information as arguments then call the sync methods on the Syncer object. Finally, call syncer_obj.sync().
```

Comment:
Should we define Dataset, Model, ModelConfig, and ModelMetrics in the previous section where we define project, experiment, experiment run.